test/e2e/specs/wp-calypso-gutenberg-social-icons-block-spec.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import { By } from 'selenium-webdriver';
+
+/**
+ * Internal dependencies
+ */
+import LoginFlow from '../lib/flows/login-flow.js';
+
+import GutenbergEditorComponent from '../lib/gutenberg/gutenberg-editor-component';
+
+import * as driverManager from '../lib/driver-manager';
+import * as driverHelper from '../lib/driver-helper';
+import * as mediaHelper from '../lib/media-helper';
+import * as dataHelper from '../lib/data-helper';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+const host = dataHelper.getJetpackHost();
+const gutenbergUser =
+	process.env.GUTENBERG_EDGE === 'true' ? 'gutenbergSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
+
+describe( `[${ host }] Calypso Gutenberg Editor: Social Icons Block (${ screenSize })`, function () {
+	this.timeout( mochaTimeOut );
+	let driver;
+
+	before( 'Start browser', async function () {
+		this.timeout( startBrowserTimeoutMS );
+		driver = await driverManager.startBrowser();
+	} );
+
+	describe( 'Preview and Publish a Post with a Social Icons block', function () {
+		let fileDetails;
+
+		// Create image file for upload
+		before( async function () {
+			fileDetails = await mediaHelper.createFile();
+			return fileDetails;
+		} );
+
+		step( 'Can log in', async function () {
+			this.loginFlow = new LoginFlow( driver, gutenbergUser );
+			return await this.loginFlow.loginAndStartNewPost( null, true );
+		} );
+
+		step( 'Can insert the Social icons block', async function () {
+			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+			await gEditorComponent.addBlock( 'Social icons' );
+
+			return await driverHelper.waitTillPresentAndDisplayed(
+				driver,
+				By.css( '.wp-block-social-icons' )
```

Comment:
Have you tried changing this selector to refer to 
`.wp-block-social-links`?

`driverHelper` should be able to find this element at that point, if the internal checks in `addBlock` are bypassed.

Reply:
For the actual `gEditorComponent.addBlock` method failing due to mismatch between the `Social Links` and `Social Icons` aria name, I added the following in the case-switch of function `getBlockSelectorSettings`:

```
			case 'Social Links':
				blockSettings = { ariaLabel: 'Social Icons'};
				break;
```

Reply:
Thanks! It was failing in the addBlock function before that. Combined with the Aria change, I may be able to use 'Social Links' and then use .wp-block-social-links. I can test it out during the week.

Reply:
Right, as far as I understand the execution flow:
- `addBlock` function calls the function `getBlockSelectorSettings`.
- `getBlockSelectorSettings` looks for `Social Links`, does not find anything in the case switch, so returns the standard values.
- down the line in `addBlock` where validation occurs, it looks for a block with aria label `Social Links` which doesn't exist, as you have found - because its actually called `Social Icons`.

The addition of the case statement means the appropriate `ariaLabel` is returned to pass the validation. Ideally, the block name and ariaLabel should match, but as you can see in the other case statements unfortunately we have other blocks that are mismatched as well.

Reply:
Thanks Edwin! Sorry about the delay. I tested this fix out and it works fine now. I should be able to move forward on these now.